cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/preview/DefaultPreviewManager.java

```diff
@@ -124,173 +115,58 @@
     this.secureStore = secureStore;
     this.transactionSystemClient = transactionSystemClient;
     this.previewDataDir = Paths.get(cConf.get(Constants.CFG_LOCAL_DATA_DIR), "preview").toAbsolutePath();
-    this.appInjectors = new ConcurrentHashMap<>();
     this.maxPreviews = cConf.getInt(Constants.Preview.PREVIEW_CACHE_SIZE, 10);
```

Comment:
Seems like this is not used anymore? Are we not enforcing max concurrent runs?

Reply:
As discussed offline we will enforce the max concurrent runs. will be part of separate PR.